source/accessibility/index.html.md.erb

```diff
@@ -77,28 +77,30 @@ DAC tested an example set of documentation pages to cover the different types of
 
 This Technical Documentation Template is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below:
 
-- Some code examples use a colour that has very low contrast with the background colour. This fails WCAG 2.1 success criterion 1.4.3 Contrast (minimum).
+- Some tables in content do not have table row headers when needed. This fails [WCAG 2.1A 1.3.1 success criterion (Info and Relationships)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).
```

Comment:
Should this be 'WCAG 2.1 success criterion 1.3.1 (Info and Relationships)'? Seems to be a different order to the other bullets, and there's what looks like a rogue 'A' after 2.1.

Reply:
Just to check - is it deliberate that we're linking to 'Understanding WCAG 2.1', rather than the WCAG criteria themselves (eg https://www.w3.org/TR/WCAG21/#info-and-relationships)?

Reply:
> Should this be 'WCAG 2.1 success criterion 1.3.1 (Info and Relationships)'? Seems to be a different order to the other bullets, and there's what looks like a rogue 'A' after 2.1.

will change

Reply:
> Just to check - is it deliberate that we're linking to 'Understanding WCAG 2.1', rather than the WCAG criteria themselves (eg https://www.w3.org/TR/WCAG21/#info-and-relationships)?

we've done that on other statements, like the [Pay statement](https://docs.payments.service.gov.uk/accessibility.html#non-compliance-with-the-accessibility-regulations)